src/main.rs

```diff
@@ -9,9 +9,15 @@
 // specific language governing permissions and limitations under the License.
 
 extern crate clap;
+#[macro_use] extern crate nickel;
+
+use nickel::{Nickel, HttpRouter};
 
 use clap::{App, Arg, SubCommand, AppSettings};
 
+use std::u16;
+use std::str::FromStr;
```

Comment:
Actually after double checking I'm not sure how to make it work in this case.  Without `use std::str::FromStr;` the call to `u16::from_str` fails.  FromStr isn't referenced directly in this file but is needed for that call to compile, so the third option doesn't seem doable here.